quad/src/qag_par.rs

```diff
@@ -455,7 +485,7 @@ mod tests {
             more_info: false,
         };
 
-        let f1 = |x: f64| vec![x.cos(), x.sin()];
+        let f1 = |x: f64| vec![x.sin() / x];
```

Comment:
Good to use $sinc$, since it's a bit less trivial (because of the irrelevant discontinuity), but shouldn't you keep at least two items to test the vector?

Of course, you could also split and have two separate tests.

Reply:
Ah, I see, you needed something with a definite integral over the whole real axis.

Another simple function to integrate is of course $e^{-|x|}$, of course :)

Reply:
sinc is an exemple of a function that fails thanks to the new implemented error flags ( as in scipy.quad, scipy.quad_vec instead continues until the max iteration error).

Reply:
Ah, ok